src/ptf/packet.py

```diff
@@ -16,7 +16,7 @@
     import scapy.layers.dhcp
     import scapy.packet
     import scapy.main
-    if not config["disable_ipv6"]:
+    if not config.has_key("disable_ipv6"):
```

Comment:
I think `if "disable_ipv6" not in config` is more readable
Also, is there a reason for the change? Isn't this key always defined in the config dictionary?

Reply:
If you just import packet (or mask), e.g.,`from ptf.mask import Mask`, the code fails with KeyError not found. I did not see a place where it is mandatory to include some top level file.